src/js/experiments/signupLinksCta.js

```diff
@@ -0,0 +1,42 @@
+const isFirstStepsPage = window.location.pathname == '/docs/2.0/first-steps/';
+
+function handleGithubDropdownClick(){
+  const dropdown = $('.gh-signup-dropdown')
+  // toggle the popup
+  $('.gh-dropdown-button').click((e) => {
+    dropdown.addClass("show");
+    e.stopPropagation()
+  })
+
+  // clicking on body closes pop if it is open.
+  $('body').click((e) => {
+    dropdown.removeClass("show");
+  })
+}
+
+/**
+ * query all signup buttons by class
+ * add click listener to track to amplitude before navigating to new page
+ * */
+function handleClickedLink() {
+  $(".track-signup-link").each(function() {
+    $(this).click((e) => {
+      window.AnalyticsClient.trackAction("dd_first-steps-signup-cta_test", {
```

Comment:
Should we include 'track-signup-link' to the drop down links? or does this account for those already